wpilibNewCommands/src/main/native/cpp/frc2/command/ParallelCommandGroup.cpp

```diff
@@ -88,3 +90,12 @@ void ParallelCommandGroup::AddCommands(
     }
   }
 }
+
+void ParallelCommandGroup::InitSendable(wpi::SendableBuilder& builder) {
+  Command::InitSendable(builder);
+  std::vector<std::string> names;
```

Comment:
You might toss in a `reserve` here since we know the count ahead of time
(applies to multiple places in the PR)